loader.cpp

```diff
@@ -116,7 +116,7 @@ wibo::Executable::~Executable() {
 	}
 }
 
-bool wibo::Executable::loadPE(FILE *file) {
+bool wibo::Executable::loadPE(FILE *file, bool exec) {
```

Comment:
it's confusing to have two things named executable here, should one of them be renamed? alternatively a comment would be good

Reply:
I added a doc comment for now, but open to better names. I'm not sure what else to call it, exactly.